Makefile

```diff
@@ -20,6 +20,9 @@ publish-core:
 publish-telem:
 	export IS_LOCAL_DEVELOPMENT=false; ./gradlew :libtelemetry:uploadArchives
 
+publish-lite:
+    export IS_LOCAL_DEVELOPMENT=false; ./gradlew :libtelemetry:uploadArchives
```

Comment:
does it look like it might be a dup of the above?

Reply:
see above